src/assets/javascripts/sw.js

```diff
@@ -0,0 +1,34 @@
+const VERSION = "v2.4"
+const APP_STATIC_RESOURCES = [
+  "/",
+  "/static/stylesheets/bootstrap.min.css",
+  "/static/stylesheets/app.css",
+  "/static/graphicarts/favicon.svg",
+  "/static/graphicarts/favicon.png",
```

Comment:
are these guaranteed to work with the custom `-base` parameter set?

Reply:
I did not test that case, but it does look like relative links should be preferred.